src/engine/bank05.asm

```diff
@@ -1765,8 +1765,97 @@ CheckIfAnyCardIDinLocation: ; 157a3 (5:57a3)
 	ret
 ; 0x157c6
 
-Func_157c6: ; 157c6 (5:57c6)
-	INCROM $157c6, $158b2
+; outputs in a total number of energy cards in hand
+; plus all the cards attached in Turn Duelist's Play Area.
+CountEnergyCardsInHandAndAttached: ; 157c6 (5:57c6)
```

Comment:
Minor nitpick, but I would add to the header where the output is returned (a and wTempAI), and which player's hand this applies to (I think since it uses CreateEnergyCardListFromHand and for context is opponent only). In the meantime, it seems that CreateEnergyCardListFromHand is also missing this information. Maybe it's evident due to being in the AI bank, but I don't think it would hurt, especially considering they may be other functions that despite also being AI-related, might still support both players depending on who's the turn holder, for example.

Reply:
You are right, I will commit some general header clean up to systematise them into a consistent format. I already got some header comments in that format, so might as well apply to all that I can.